cargo-shuttle/src/lib.rs

```diff
@@ -606,6 +608,14 @@ impl Shuttle {
             addr
         );
 
+        println!("
+    To host the application on Shuttle run
+       `cargo shuttle project start`
+    if project environment is not created previously and then run
+       `cargo shuttle deploy`
+    Your application will be hosted at https://{}.shuttleapp.rs
+", service_name);
```

Comment:
I'm not sure we need this message in local run, but it might be useful :thinking: Either way we should not show the URL the project will be hosted at here, because they may use a name that is already taken, but that won't fail in local runs, it will only fail when they run `cargo shuttle project start`.

Reply:
The users might do the local run multiple time, over and over again; do you think this message will get annoying to see every time? If then then we should remove this but the user already see a lot of output from cargo, we could do however print this to stderr, but there are no `eprintln` in the entire file just `println`.

Reply:
There is also the issue of possibly printing this multiple times because locally we'll spin a runtime for each workspace service. This log might be handy for people, so maybe we can add it in the `local_run` function, but before waiting for runtimes to finish. This is somehow conflicting with what I said previously, but I see the point here of having the next logical command be `project start`/`deploy`.

Reply:
`local_run` both of unix and windows call `spin_local_runtime` and I have added those two suggestion in this method rather than adding it to both `local_run` methods. Anything wrong with this?

Reply:
oops sorry `spin_local_runtime` gets called multiple times if we have multiple services, I didn't see this because of my limited testing on hello world shuttle service.